From 8797adc1c6e84789c261ee24afe5a1cbfaddba6b Mon Sep 17 00:00:00 2001
From: Denys Vlasenko <vda.linux@googlemail.com>
Date: Mon, 31 Dec 2018 19:50:06 +0100
Subject: [PATCH 06/84] bc: remove superfluous assigment

Signed-off-by: Denys Vlasenko <vda.linux@googlemail.com>
---
 miscutils/bc.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/miscutils/bc.c b/miscutils/bc.c
index 798bc0a3e..febf51cfd 100644
--- a/miscutils/bc.c
+++ b/miscutils/bc.c
@@ -2642,7 +2642,6 @@ static void bc_num_parseBase(BcNum *n, const char *val, unsigned base_t)
 	BcNum base;
 	BcDig temp_digs[ULONG_NUM_BUFSIZE];
 	BcDig base_digs[ULONG_NUM_BUFSIZE];
-	BcDig c = '\0';
 	size_t digits;
 
 	bc_num_init_DEF_SIZE(&mult);
@@ -2657,6 +2656,7 @@ static void bc_num_parseBase(BcNum *n, const char *val, unsigned base_t)
 
 	for (;;) {
 		unsigned v;
+		char c;
 
 		c = *val++;
 		if (c == '\0') goto int_err;
@@ -2679,6 +2679,7 @@ static void bc_num_parseBase(BcNum *n, const char *val, unsigned base_t)
 	digits = 0;
 	for (;;) {
 		unsigned v;
+		char c;
 
 		c = *val++;
 		if (c == '\0') break;
-- 
2.16.2

